src/app/components/home/clip/clip.component.html

```diff
@@ -145,6 +145,7 @@
     [ngStyle]="{ 'height': imgHeight + 'px' }"
     (click)="videoClick.emit($event)"
     class="screencap"
+    [volume]="shouldMute()"
     onmouseover="this.muted = false;"
```

Comment:
Might make more sense to have
```
if (!this.forceMute) { this.muted = false };
...
```

Thoughts? 😄

Reply:
What about just bringing the `shouldMute()` method right into the template?
```
[volume]="forceMute ? 0 : 1"
```
If I remember right I had trouble with muting -- since hovering over the element toggles the `muted` attribute, hence why I resorted to the hack with the `volume` property 😅

Reply:
Well I was more thinking it might be better to disable the audio on hover behaviour, rather than also using the `volume` property? There's no point unmuting it if the volume is 0 anyway, but maybe your approach has less edge cases... 🤔